docs/sources/logql/query_examples.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Query examples
+weight: 40
+---
+
+# Query examples
+
+Some useful query examples here.
+
+## Log Query examples
+
+### Examples that filter on IP address 
+
+- Return log lines that are not within a range of IPv4 addresses:
+
+    ```logql
+    {job_name="myapp"} != ip("192.168.4.5-192.168.4.20")
+    ```
+
+
+- This example matches log lines with all IPv4 subnet values `192.168.4.5/16` except IP address `192.168.4.2`:
+
+    ```logql
+    {job_name="myapp"}
+		| logfmt
+		| addr = ip("192.168.4.5/16")
+		| addr != ip("192.168.4.2")
+    ```
+
+### Examples that aid in security evaluation
+
+- Extract the user and IP address of failed logins from Linux `/var/log/secure`
+
+    ```logql
+    {job="security"} 
+        |~ "Invalid user.*"
+        |regexp "(^(?P<user>\\S+ {1,2}){8})"
+        |regexp "(^(?P<ip>\\S+ {1,2}){10})"
```

Comment:
Just a nit: I like putting a space after the pipe. You have added it for some of them. I think we should add them here too?
```suggestion
        | regexp "(^(?P<user>\\S+ {1,2}){8})"
        | regexp "(^(?P<ip>\\S+ {1,2}){10})"
```